ocs_ci/ocs/node.py

```diff
@@ -1107,3 +1107,63 @@ def is_node_labeled(node_name, label=constants.OPERATOR_NODE_LABEL):
     """
     node_names_with_label = machine.get_labeled_nodes(label=label)
     return node_name in node_names_with_label
+
+
+def check_taint_on_nodes(taint=None):
+    """
+    Function to check for particular taint on nodes
+
+    Args:
+        taint (str): The taint to check on nodes
+
+    """
+    taint = taint if taint else constants.OPERATOR_NODE_TAINT
+    ocs_nodes = get_ocs_nodes()
+    flag = -1
+    for node_obj in ocs_nodes:
+        if node_obj.get().get("spec").get("taints"):
+            if taint in node_obj.get().get("spec").get("taints")[0].get("key"):
+                log.info(f"Node {node_obj.name} has taint {taint}")
+                flag = 1
+        else:
+            flag = 0
+        return bool(flag)
+
+
+def taint_nodes(nodes_to_taint=None):
+    """
+    Function to taint nodes
```

Comment:
This looks so generic, the function is tainting only with "node.ocs.openshift.io/storage=true:NoSchedule"
Please update the doc string